blocks/v2-feature-carousel/v2-feature-carousel.js

```diff
@@ -0,0 +1,95 @@
+import {
+  createElement,
+  unwrapDivs,
+  adjustPretitle,
+} from '../../scripts/common.js';
+import {
+  listenScroll,
+  createArrowControls,
+} from '../../scripts/carousel-helper.js';
+
+const blockName = 'v2-feature-carousel';
+
+const updateActiveClass = (elements, entry) => {
+  elements.forEach((el, index) => {
+    if (el === entry.target) {
+      el.classList.add('active');
+      let arrowControl = document.querySelector(`.${blockName}__button:disabled`);
```

Comment:
With `document` it can affect other `v2-feature-carousel`. We should limit it to the block only.